packages/mds-ingest-service/@types/index.ts

```diff
@@ -199,6 +199,20 @@ export interface TelemetryLabel {
   telemetry_charge: Nullable<number>
 }
 
+export interface TelemetryAnnotationDomainModel {
+  device_id: UUID
+  provider_id: UUID
+  timestamp: Timestamp
+  res_08: string
```

Comment:
Maybe add an `h3` prefix to these?

Reply:
Ditto, I think that a sensible naming scheme would be `h3_${cell_size}` e.g. `h3_13`, `h3_12`, etc...